src/mame/drivers/rockrage.cpp

```diff
@@ -152,7 +152,9 @@ static INPUT_PORTS_START( rockrage )
 	PORT_DIPSETTING(    0x02, "2" )
 	PORT_DIPSETTING(    0x01, "3" )
 	PORT_DIPSETTING(    0x00, "5" )
-	PORT_DIPUNUSED_DIPLOC( 0x04, IP_ACTIVE_LOW, "SW2:3" )
+	PORT_DIPNAME( 0x04, 0x00, "Sound Adjustment" )      PORT_DIPLOCATION("SW2:3")
```

Comment:
Maybe this should just be `DEF_STR( Cabinet )` instead of `"Sound Adjustment"`. The Japanese in the manual says "Monitor and Sound Switch" and poor translations are pretty common in these old things.

That being said, I couldn't tell whether this switch has any effect. SW2:3 is often the Cabinet switch for other Konami hardware, but maybe the software ignores it here? Though it _is_ strange that both the Japanese and English descriptions of this Cabinet switch in the Rock'n Rage manual differ from all the other Konami manuals AFAICT (they tend to say "Upright/Table Switch").